src/constants.ts

```diff
@@ -66,8 +66,8 @@ export class FileNames {
 }
 
 export enum OperationType {
-  Compile = 'Compile device code',
-  Upload = 'Upload device code'
+  Compile = 'Device code compilation',
```

Comment:
Is it better to express as verb+noun form? e.g. Compile device  code?

Reply:
The value is used for output. Now we have information message like `Device code compilation operation failed. Please do something.` That's why I change the string value to be a noun. It makes grammatical mistakes to say `Compile device code operation failed.`